tools/cbmc/proofs/make_proof_makefiles.py

```diff
@@ -139,7 +144,12 @@ def dump_makefile(dyr, system):
             makefile[name] = " ".join(new_value)
         else:
             makefile[name] = compute(value, so_far, system, name, dyr)
-
+    
+    if (("EXPECTED" not in makefile.keys()) or
+            makefile["EXPECTED"].lower() == "true"):
```

Comment:
Over here you're checking for `"true"`, but on [line 148 of `make_configuration_directories.py`](https://github.com/aws/amazon-freertos/pull/653/files#diff-db83c98f72f65488a4d3ad843aee5664R148) you're dumping `True`. Is this a mistake?

Reply:
In line 148, I use a boolean primitive. So it must be a capital `T` there.

Before reaching this line, `compute()` is called which does a string cast in [L178 of make_proof_makefiles.py](https://github.com/aws/amazon-freertos/blob/a3b46a51f6c276e00c32b5ca3f9223420fd1737b/tools/cbmc/proofs/make_proof_makefiles.py#L178)

That is the magic, why it work here. I will add explicit string cast to point it out.